src/modules/organizations/components/organization-projects-list.jsx

```diff
@@ -9,15 +9,28 @@ const OrganizationProjectsList = ({ projects, onRemove }) => {
   }
 
   return (
-    <ResourcesList
-      resources={projects}
-      resourceType={'projects'}
-      showOwnerName={true}
-      showRemove={true}
-      onRemove={onRemove}
-      showStatus={true}
-      title={'Project List'}
-    />
+    <div>
+      <h3 className="organization-layout__section-header">Affiliated Projects</h3>
+      <ul className="organization-layout__section-instructions">
+        <li className="organization-layout__section-instructions--list-items">
+          If you see CONTACT ORGANIZATION OWNER as the owner, contact the organization owner,
+          or collaborators, to get access to this project.
+        </li>
+        <li className="organization-layout__section-instructions--list-items">The visibility of each project is displayed to the right of the project&apos;s owner.</li>
+        <li className="organization-layout__section-instructions--list-items">Only the owner and collaborators can view a NOT PUBLICLY VISIBILE project.</li>
+        <li className="organization-layout__section-instructions--list-items">Anyone with the URL can access a PUBLICLY VISIBILE project.</li>
```

Comment:
This is misleading, see related comment in `resources-list`, I think confusing project attributes of `private` vs `launched_approved`?